src/app/core/components/thread/thread.component.ts

```diff
@@ -165,7 +170,19 @@ export class ThreadComponent implements AfterViewInit, OnDestroy {
     );
   }
 
-  changeThreadStatus(): void {
-    this.actionFeedbackService.error($localize`Not implemented`);
+  changeThreadStatus(open: boolean, threadInfo: ThreadInfo): void {
+    this.updateThreadService.update(threadInfo.itemId, threadInfo.participantId, open ? {
+      status: 'waiting_for_trainer',
+      helperGroupId: environment.allUsersGroupId,
+    } : { status: 'closed' }).subscribe({
+      next: () => this.threadService.refresh(),
+      error: error => {
+        if (errorIsHTTPForbidden(error)) {
+          this.actionFeedbackService.error($localize`Unable to open the thread`);
+          return;
+        }
```

Comment:
Actually, as we check all possible error and disable button if the action is not possible, this block is not useful. All errors are "unexpectedError" errors.

Reply:
Fixed